pipeline/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDaoTest.scala

```diff
@@ -35,56 +37,124 @@ class IdentifiersDaoTest
       testWith((identifiersDao, identifiersTable))
     }
 
-  describe("lookupID") {
-    it("gets an Identifier if it finds a matching SourceSystem and SourceId") {
+  describe("lookupIds") {
+
+    it("retrieves a single id from the identifiers database") {
```

Comment:
minor: more accurately, what this tests is the stronger property that it can store and then retrieve an identifier from the DB (and the old description was wrong too)

The pattern we have in the storage service/scala-storage library would be closer to

```scala
withIdentifiersDao(existingEntries) { ... }
```

and when the caller defines that fixture, they're responsible for putting those entries in place – in this case that would be SQL statements, like there are for testing the "does it store a thing correctly" code below.